### ISSUE TITLE
[MSFT-213] User only can focus on "Username/ Password" and can't focus on other items using tab

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2875]
**Steps to Reproduce**
1. Go to: https://accounts.topcoder.com/member 
2. Use the tab key to focus on individual items in page

**Expected Result**
User should be able to select all items using tab

**Actual Result**
User only can focus on "Username/ Password" and can't focus on other items using tab

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F22

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096248 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096248 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096248 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by r0hit-gupta** [comment]
https://github.com/appirio-tech/accounts-app/pull/243

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2875 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2875 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2875 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097056 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097056 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by mishacucicea** [comment]
Duplicate of #2800

**Comment by Oanh-and-only-Oanh** [comment]
closing because it's a duplicate bug
